cobbler/modules/managers/__init__.py

```diff
@@ -82,6 +82,26 @@ def sync(self) -> int:
         self.write_configs()
         return self.restart_service()
 
+    def sync_single_system(self, system: "System") -> int:
+        """
+        This synchronizes data for a single system. The default implementation is
+        to trigger full synchronization. Manager modules can overwrite this method
+        to improve performance.
+
+        :param system: A system to be added.
+        """
+        del system
```

Comment:
Is this deletion really necessary here? (as we are adding a new system here)

Reply:
All of these remove unused pointers to arguments, based on https://google.github.io/styleguide/pyguide.html

Reply:
In that case, a comment should explain why the deletion is done:
> Unused argument warnings can be suppressed by deleting the variables at the beginning of the function. Always include a comment explaining why you are deleting it. “Unused.” is sufficient.

Reply:
Is this discussion resolved or is there something outstanding that w want to do?